Formula/gitlab-ci-multi-runner.rb

```diff
@@ -41,21 +48,24 @@ def install
     end
 
     cd dir do
-      resource("prebuilt-x86_64.tar.xz").stage do
-        system "go-bindata", "-pkg", "docker", "-nocompress", "-nomemcopy",
-                             "-nometadata", "-o",
-                             "#{dir}/executors/docker/bindata.go",
-                             "prebuilt-x86_64.tar.xz"
+      ["prebuilt-x86_64.tar.xz", "prebuilt-arm.tar.xz"].each do |r|
```

Comment:
I don't know how Homebrew works internally, so maybe my comment is invalid, but from what I see here it looks that the `go-bindata -pkg ...` command will be executed twice: first time with `prebuilt-x86_64.tar.xz` as the argument and second time with `prebuilt-arm.tar.xz` as the argument. If yes, then the second execution will overwrite the `bindata.go` file. And that's not the goal which we want to achieve.

It should be a one execution of this command with both files as the final arguments, so something looking like:
```
go-bindata -pkg docker \
           -nocompress \
           -nomemcopy \
           -nometadata \
           -o executors/docker/bindata.go
           prebuilt-x86_64.tar.xz
           prebuilt-arm.tar.xz
```

With this both files will be available for Docker executor as assets stored internally in `bindata.go` file.